misc/python/materialize/mzcompose/services.py

```diff
@@ -52,6 +52,7 @@
     "enable_ld_rbac_checks": "true",
     "enable_rbac_checks": "true",
     "enable_monotonic_oneshot_selects": "true",
+    "enable_try_parse_monotonic_iso8601_timestamp": "true",
```

Comment:
oh wow, did I totally brain fart on https://github.com/MaterializeInc/materialize/pull/20414#issuecomment-1638367395? 🤦‍♂️ sorry about that!

Reply:
no worries 😀